runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -249,7 +249,7 @@ static void reportHookDetail(J9VMThread *curThread, char *name, char *format, ..
    if (TR::Options::getCmdLineOptions()->getVerboseOption(TR_VerboseHookDetails))
       {
       TR_VerboseLog::vlogAcquire();
-      TR_VerboseLog::writeLine(TR_Vlog_HD," %x: ", (int)(intptr_t)curThread);
+      TR_VerboseLog::writeLine(TR_Vlog_HD,"vmThread=%p hook %d detail ", curThread, name);
```

Comment:
Should the format specifier for `name` be `%s` ?

Reply:
Wow that's a bad mistake by me. Fixed in [force-push](https://github.com/eclipse/openj9/compare/b29d3904e45d3e150277c58d817082c7529d0202..98d8b27d25831195ee3b78451e14782806c4f7f7). Thanks for catching this.